modules/projects/client/views/project-partials/project-tombstone.html

```diff
@@ -48,7 +48,7 @@ <h2>Project Details</h2>
 			</div>
 			<div class="col-sm-4 col-md-4" ng-if="project.eacDecision">
 				<div class="detail-row">
-					<label>EAC Decision</label>
+					<label>EA Decision</label>
```

Comment:
Should this actually be "EA" and not "EAC"?

Looks like all of the variables use "eac".

Reply:
The ticket says to update the display label from "EAC Decision" to "EA Decision"

Reply:
Oh gotcha.  Thats fair then.